backend/apps/progress/services/milestone_track_service.py

```
@@ -0,0 +1,193 @@
+        for season in active_seasons:
+            completed_ids = UserMilestoneCompletion.objects.filter(
+                user=user, milestone__season=season
+            ).values_list("milestone_id", flat=True)
+
+            incomplete_milestones = season.milestones.exclude(id__in=completed_ids).order_by("target_value")
+
+            for milestone in incomplete_milestones:
+                progress = MilestoneTrackService.calculate_progress(user, milestone)
+                if progress >= milestone.target_value:
+                    with transaction.atomic():
+                        completion, created = UserMilestoneCompletion.objects.select_for_update().get_or_create(
```

Ruff (0.15.21)

[warning] Unpacked variable `completion` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)